java/edit-and-sync-features/README.md

```diff
@@ -1,39 +1,40 @@
 # Edit and sync features
+
 Synchronize offline edits with a feature service.
 
-![Edit and sync features App](edit-and-sync-features.png)
+![Image of edit and sync features](edit-and-sync-features.png)
+
+## Use case
+
+A survey worker who works in an area without an internet connection could take a geodatabase of survey features offline at their office, make edits and add new features to the offline geodatabase in the field, and sync the updates with the online feature service after returning to the office.
 
 ## How to use the sample
-1. Pan and zoom into the desired area, making sure the area you want to take offline is within the current extent of the `MapView`. 
-1. Tap on the Generate Geodatabase button. This will call `generateGeodatabase()`, which will return a `GenerateGeodatabaseJob`. 
-1. Once the job completes successfully, a `GeodatabaseFeatureTable` and a `FeatureLayer` are created from the resulting `Geodatabase`. The `FeatureLayer` is then added to the `ArcGISMap`. 
-1. Once the `FeatureLayer` generated from the local `Geodatabase` is displayed, a `Feature` can be selected by tapping on it. The selected `Feature` can be moved to a new location by tapping anywhere on the map. 
-1. Once a successful edit has been made to the `FeatureLayer`, the Sync Geodatabase button is enabled. This button synchronizes local edits made to the local `GeoDatabase` with the remote feature service using `syncGeodatabase()` which generates `SyncGeodatbaseParameters` and passes them to a `SyncGeodatabaseJob`.
-1. Once the job successfully completes, the local edits are synchronized with the feature service.
+
+Pan and zoom into the desired area, making sure the area you want to take offline is within the current extent of the `MapView`. Tap on the Generate Geodatabase button to take the area offline. When complete, the map will update with a red outline around the offline area. To edit features, tap to select a feature, and tap again anywhere else on the map to move the selected feature to the tapped location. To sync the edits with the feature service, click the "Sync geodatabase" button.
```

Comment:
we shouldn't reference direct API classes within this section: we can just refer to `MapView` as map view.

Also, let's try and remember to put "" around button descriptions -> "Generate Geodatabase" for readability

Reply:
will try to remember both of these!